widgetssdk/src/main/java/com/glia/widgets/core/chathead/ChatHeadService.kt

```diff
@@ -101,17 +111,14 @@ class ChatHeadService : Service() {
     private fun initChatHeadPosition(
         params: WindowManager.LayoutParams, chatHeadPosition: Pair<Int, Int>
     ) {
-        val displayMetrics = displayMetrics
-        val screenHeight = displayMetrics.heightPixels
-        val screenWidth = displayMetrics.widthPixels
-        params.x = notNullOrDefault(chatHeadPosition.first, getDefaultXPosition(screenWidth))
-        params.y = notNullOrDefault(chatHeadPosition.second, getDefaultYPosition(screenHeight))
+        val display = displaySize
+        params.x = chatHeadPosition.first ?: getDefaultXPosition(display.width)
+        params.y = chatHeadPosition.second ?: getDefaultYPosition(display.height)
     }
 
-    private fun notNullOrDefault(item: Int?, defaultItem: Int): Int = item ?: defaultItem
-
     private fun getDefaultXPosition(screenWidth: Int): Int = screenWidth - chatHeadSize - chatHeadMargin
 
+    //🤔 Some indescribable magic here 🤦‍
```

Comment:
Let me double-check because it's a public repository. Have you added this comment intentionally? I don't mind but just want to make sure.

Reply:
Yes, because I haven't able to understand or even guess what is it, why `10*8`.
But if it's not ok I can remove.

Reply:
I think it's not 10*8 but 8/10 of the entire height. Or, if simplify, it's 4/5 of the entire height. But why exactly 4/5 - don't know. Maybe just a strict position.

Reply:
It seems it's better to leave this method without any comment)